sound/soc/intel/boards/sof_sdw.c

```diff
@@ -555,6 +555,7 @@ static struct sof_sdw_codec_info codec_info_list[] = {
 				.direction = {true, true},
 				.dai_name = "rt700-aif1",
 				.dai_type = SOF_SDW_DAI_TYPE_JACK,
+				.dailink = {0, 1},
```

Comment:
can those 0,1,4, etc be replaced by macros?

Reply:
> can those 0,1,4, etc be replaced by macros?

Yes, will do.